spec/lib/open_food_network/feature_toggle_spec.rb

```diff
@@ -15,4 +15,21 @@
       expect(feature_toggle.enabled?(:foo)).to be true
     end
   end
+
+  describe ".setup!" do
+    it "created all current features at boot time" do
+      expect(Flipper.features.count)
+        .to eq feature_toggle::CURRENT_FEATURES.count
```

Comment:
Perhaps we can map and sort these to arrays that could be compared?

Reply:
Great idea, added.